docs/lockfile_generation.md

```diff
@@ -34,18 +30,22 @@ The Phylum CLI can generate a lockfile when it is given a manifest file.
 [go]: https://go.dev
 [cargo]: https://www.rust-lang.org
 
+:::tip
+
 For files that can be handled by multiple generators, a fallback is used:
 
 * `package.json` will use `npm`
 * `pyproject.toml` will use `pip`
 
 This can be overridden on the command line with the `--lockfile-type` (`-t`) option. For example:
 
-```
+```sh
 phylum analyze -t yarn package.json
 ```
 
-### Lockfile detection
+:::
```

Comment:
I'd assume this is similar to the `:::tip` thing?

Reply:
Yes...that is the closing identifier for a [Docusaurus admonition](https://docusaurus.io/docs/markdown-features/admonitions)